playbooks/runcible.yml

```diff
@@ -0,0 +1,105 @@
+---
+- hosts: all
+  become: true
+  roles:
+    - puppet_repositories
+    - pulp_repositories
+    - epel_repositories
+
+- hosts: all
+  tasks:
+    - name: 'Add OSTree repo'
+      yum_repository:
+        name: atomic7-testing
+        description: "Atomic testing repository"
+        baseurl: http://cbs.centos.org/repos/atomic7-testing/x86_64/os/
+        gpgcheck: no
+        enabled: yes
+      become: true
+
+    - name: 'Install packages'
+      yum:
+        name: "{{ item }}"
+        state: "present"
+        update_cache: true
+      become: true
+      with_items:
+        - ruby-devel
+        - rubygem-bundler
+        - libxml2-devel
+        - ruby
+        - rake
+        - gcc
+        - wget
+        - git
+        - nss
+        - puppet
+        - glib2
+        - ostree
+
+    - name: "Install RVM gpg key"
+      shell: "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
+
+    - name: "Install RVM"
```

Comment:
Is RVM actually used? I don't see anything activating it.

Reply:
Aye it is, bet deceptive perhaps but without this it does not work.

Reply:
I'm guessing it installs something into the bash login profile and ansible uses that implicitly.